example.tex

```diff
@@ -273,6 +273,7 @@ \chapter{Colors}
     |DmgCoral|                      & Orange-pink used in DMG Part 2 \\
     |DmgSlateGray| (|DmgSlateGrey|) & Blue-gray used in PHB Part 3 \\
     |DmgLilac|                      & Purple-gray used in DMG appendix \\
+    |BrGreen|                       & Gray-green used for tables in Basic Rules\\
```

Comment:
How about making this table a regular `tabularx` and using `\rowcolor` to set each row to the corresponding color?

Reply:
Sure.